community-packs.json

```diff
@@ -88,5 +88,15 @@
         "versions_url": "https://raw.githubusercontent.com/lurch9229/Ape-Escape-Tracker/versions/versions.json",
         "icon_url": "https://raw.githubusercontent.com/lurch9229/Ape-Escape-Tracker/refs/heads/main/images/icons/specterIcon.png",
         "description":  "Ape Escape autotracker pack for PopTracker"
+    },
+    {
+    "AP_Man_Xeno1_Xan": {
+        "name": "Xenoblade Chronicles Manual PopTracker",
+        "author": "skipperki1",
+        "platform": "poptracker",
```

Comment:
I don't know if there is a difference between the various versions, but this should probably be "multiple" if the same pack works for the various platforms? Or maybe "archipelago" if this pack only makes sense in combination with Archipelago, but does not care which game version you use.